interfaces/builtin/pulseaudio.go

```diff
@@ -20,30 +20,170 @@
 package builtin
 
 import (
+	"bytes"
+
 	"github.com/snapcore/snapd/interfaces"
+	"github.com/snapcore/snapd/release"
 )
 
-const pulseaudioConnectedPlugAppArmor = `
+var pulseaudioConnectedPlugAppArmor = []byte(`
+capability dac_override,
+
+/etc/machine-id r,
+
+/{run,dev}/shm/pulse-shm-* rwk,
+
+# Running as system instance
```

Comment:
I think this comment is not quite right as it applies to both when running as a snap and when running on the system, no?